File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/secure3d/ui/Secure3DViewComponent.java
package com.fintechplatform.ui.secure3d.ui;

import javax.inject.Singleton;

import dagger.Component;

@Singleton
@Component(modules = {
        Secure3DPresenterModule.class }
)
public interface Secure3DViewComponent {
    void inject(Secure3DFragment fragment);
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/secure3d/di/Secure3DViewComponent.kt
package com.fintechplatform.ui.secure3d.di

import com.fintechplatform.ui.secure3d.Secure3DFragment
import dagger.Component
import javax.inject.Singleton

@Singleton
@Component(modules = [Secure3DPresenterModule::class])
interface Secure3DViewComponent {
    fun inject(fragment: Secure3DFragment)
}